commercial-support.tt

```diff
@@ -12,6 +12,7 @@
 <p>The list is sorted in alphabetical order.</p>
 
 <ul>
+  <li><a href="https://blackriversoft.com/">Black River Software</a></li>
```

Comment:
It appears the site is down? :grimacing:

Reply:
Oh, I don't have HTTPS

Reply:
Fixed

Reply:
Down for me as well.

Reply:
It shows with curl 🤔